FrontEnd/src/components/SignUp/components/signup/signup-form/SignUpFormContent.js

```diff
@@ -1,7 +1,25 @@
+import { useState } from "react";
 import { useForm } from "react-hook-form";
+import EyeInvisible from "../../../../authorization/EyeInvisible";
+import EyeVisible from "../../../../authorization/EyeVisible";
 import styles from "./SignUpFormContent.module.css";
 
 export function SignUpFormContentComponent() {
+  const [typePassword, setTypePassword] = useState("password")
```

Comment:
It would be mich more easier to use `boolean` here and call it `showPassword`

Reply:
Changed. Thank you.